go/client/chat_cli_rendering.go

```diff
@@ -127,21 +127,29 @@ func (f messageFormatter) renderAuthorAndTime() string {
 }
 
 func (f messageFormatter) body() []string {
-	bodies := keybase1.Message(f).MessagePlaintext.MessageBodies
-	if len(bodies) == 0 {
-		return nil
-	}
-	typ, err := bodies[0].MessageType()
+	version, err := f.MessagePlaintext.Version()
 	if err != nil {
+		// XXX would like to log this error or return it...
```

Comment:
i was just refactoring this cli code as little as possible, and messageFormatter is not contextified, it's a wrapper around an avdl type.